server/index.js

```diff
@@ -54,6 +54,103 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  if (!req.session.cartId) {
+    return res.status(400).json([]);
```

Comment:
This is actually not an error scenario. No items is an `OK` situation so `200` is fine.